src/deploy/functions/runtimes/node/triggerParser.js

```diff
@@ -7,7 +7,28 @@ var EXIT = function () {
   process.exit(0);
 };
 
-(function () {
+/**
+ * Dynamically load import function to prevent TypeScript from
+ * transpiling into a require.
+ *
+ * See https://github.com/microsoft/TypeScript/issues/43329.
+ */
+// eslint-disable-next-line @typescript-eslint/no-implied-eval
+var dynamicImport = new Function("modulePath", "return import(modulePath)");
```

Comment:
Prefer const

Reply:
Q: Does it matter that this script, for some reason, uses "old" JS style? (e.g. uses `var` pervasively instead of `let` or `const`). I thought it had to do with `const` not being supported on old NodeJS version, but it looks like `const` is supported from NodeJS 6 and up.